sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -499,6 +506,53 @@ case class DescribeTableCommand(table: TableIdentifier, isExtended: Boolean, isF
     }
   }
 
+  private def describeDetailPartitionInfo(
```

Comment:
`describeDetailPartitionInfo ` -> `describeDetailedPartitionInfo`

`detail` is not an adjective. I found you use it in multiple function names. Maybe correct them. Thanks!